src/include/daos_cont.h

```diff
@@ -24,6 +24,7 @@ extern "C" {
 #endif
 
 #include <daos_security.h>
+#include <daos_types.h>
```

Comment:
Discussed with @mchaarawi will get rid of this #include and rely on #include <daos.h> include order being correct